src/tools/dev/scripts/bv_support/bv_visit.sh

```diff
@@ -345,6 +345,15 @@ function build_visit
         FEATURES="${FEATURES} -DCPACK_RPM_SPEC_MORE_DEFINE:STRING=\"%global_python_bytecompile_errors_terminate_build 0\""
     fi
 
+    FEATURES="${FEATURES} -DENABLE_EXAMPLES:BOOL=OFF"
```

Comment:
Not sure we want to turn these off in this script. @brugger1, @cyrush thoughts?

Reply:
We should not turn these off. This change should be backed out.

Reply:
Oops, sorry about that those were to facilitate a local build and I neglected to take them out before the MR. I'll remove them and force push.

Reply:
done